src/core/libs/supl/CMakeLists.txt

```
@@ -55,9 +62,10 @@ target_include_directories(core_libs_supl
         ${CMAKE_CURRENT_SOURCE_DIR}
         ${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl
         ${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-rrlp
+        ${CMAKE_SOURCE_DIR}/src/core/libs/supl/types
 )
 
 set_target_properties(core_libs_supl PROPERTIES
     LINKER_LANGUAGE C
-    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR};${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl;${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-rrlp;${GNUTLS_INCLUDE_DIR}"
+    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR};${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl;${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-rrlp;${CMAKE_SOURCE_DIR}/src/core/libs/supl/types;${GNUTLS_INCLUDE_DIR}"
```

Similarly, consider
```suggestion
    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR};${CMAKE_CURRENT_SOURCE_DIR}/asn-supl;${CMAKE_CURRENT_SOURCE_DIR}/asn-rrlp;${CMAKE_CURRENT_SOURCE_DIR}/types;${GNUTLS_INCLUDE_DIR}"
```

Now, I'd have to prove this out, but I'm pretty sure that this is equivalent (and more maintainable):
```suggestion
    INTERFACE_INCLUDE_DIRECTORIES 
        ${CMAKE_CURRENT_SOURCE_DIR}
        ${CMAKE_CURRENT_SOURCE_DIR}/asn-supl
        ${CMAKE_CURRENT_SOURCE_DIR}/asn-rrlp
        ${CMAKE_CURRENT_SOURCE_DIR}/types
        ${GNUTLS_INCLUDE_DIR}
```

---

Done.